x-pack/test/functional/apps/index_management/data_streams_tab/data_streams_tab.ts

```diff
@@ -132,6 +132,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await es.indices.deleteIndexTemplate({
           name: `logsdb_index_template`,
         });
+        await browser.refresh();
```

Comment:
Is there a way to avoid the browser refreshes? I'm sure they're rather slow. Is there a refresh button? Or is it possible to navigate away and back?